src/modules/testimonials/testimonials.service.ts

```diff
@@ -50,4 +50,16 @@ export class TestimonialsService {
       }
     }
   }
+
+  async deleteTestimonial(id: string) {
+    const testimonial = await this.testimonialRepository.findOneBy({ id });
+
+    if (!testimonial)
+      throw new NotFoundException({
+        success: false,
+        message: 'Testimonial not found',
+        status_code: HttpStatus.NOT_FOUND,
+      });
+    await this.testimonialRepository.delete({ id });
```

Comment:
Please implement [onDelete option](https://orkhan.gitbook.io/typeorm/docs/relations#relation-options) in the `./entities/testimonials.entity` file.

Reply:
I don't think we'll want to delete a testimonial if the user account was deleted.

Reply:
@Heba-WebDev and @TheCodeGhinux will be able to help us. But technically we should not be displaying a user's review when they deleted.